src/yukon/backend/src/devserv/description/example/example_sys_description.json

```diff
@@ -0,0 +1,152 @@
+{
+        "bus": {
+                "resilience": 2,
+                "name": "socketcan0",
+                "protocol": "CANFD"
+        },
+        "health": {
+                "uptime": 1000,
+                "health": "All good",
+                "version": "1.2.3"
+        },
+        "nodes": {
+                "plugandplay": {
+                        "45678": "0",
+                        "98765": "1",
+                        "98766": "2",
+                        "98767": "3",
+                        "98768": "4",
+                        "45821": "5"
+                },
+                "detail": [{
+                                "id": 0,
+                                "name": "bms_772",
```

Comment:
We need the name as well, see the prior comment. Names and descriptions might be a little mixed up in this mock-up. In this specific case, the name (i.e., the field `name` of uavcan.node.GetInfo) may be `com.arkelectron.bms_772`, and the description may be like `Main battery BMS`.